src/FlatSharp.Compiler/TypeDefinitions/FieldDefinition.cs

```diff
@@ -46,81 +48,6 @@ internal class FieldDefinition
 
         public bool SharedString { get; set; }
 
-        public string GetClrTypeName(BaseSchemaMember baseMember)
```

Comment:
It's difficult for me to tell what's changed here since methods have moved around. I'm not opposed to reordering anything, but it'd be nice to do that in separate PRs.